components/input/TextArea.tsx

```diff
@@ -9,15 +9,21 @@ import { fixControlledValue, resolveOnChange } from './Input';
 export interface TextAreaProps extends RcTextAreaProps {
   allowClear?: boolean;
   bordered?: boolean;
+  hasCount?: boolean;
```

Comment:
Is better naming `showCount`?

Reply:
The name is from antd-mobile, i prefer yours or enableCount

Reply:
https://github.com/ant-design/ant-design/wiki/API-Naming-rules#props

@07akioni What do you think?

Reply:
`showCount`